utils/src/treev2/quickPickWizard/FindByIdQuickPickStep.ts

```diff
@@ -27,7 +27,7 @@ export class FindByIdQuickPickStep<TContext extends types.QuickPickWizardContext
     public async getSubWizard(wizardContext: TContext): Promise<types.IWizardOptions<TContext> | undefined> {
         // TODO: this code is nearly identical to `RecursiveQuickPickStep`, but this class can't inherit from it because it's
         // not at all based on context value for filtering
-        const lastPickedItem = getLastNode(wizardContext);
+        const lastPickedItem = getLastNode<vscode.TreeItem>(wizardContext);

```

Comment:
This isn't a `vscode.TreeItem` is it? I thought we'd only put the model objects into the pick list.

Reply:
Right, needed to get it from the `TreeDataProvider` instead.